c_sharp_subtheme.libraries.yml

```diff
@@ -87,7 +87,11 @@
 #       css/pattern-library/theme/print.css: { media: print }
 
 allpages:
-  version: VERSION
   css:
     theme:
       dist/css/main.css: { }
+  js:
+    src/js/c_sharp_subtheme.behaviors.js: { }
```

Comment:
@jdwjdwjdw Remove this as well to prevent warning messages.

Reply:
I think you're looking at an outdated version of this file - I removed it in my latest commit.

Reply:
@jdwjdwjdw You're right. I refreshed the page a few times on GitHub to make sure, but my local has it removed. Disregard.